maintainers/maintainer-list.nix

```diff
@@ -2375,6 +2375,12 @@
     github = "delan";
     githubId = 465303;
   };
+  deliciouslytyped = {
+    email = "47436522+deliciouslytyped@users.noreply.github.com";
+    github = "deliciouslytyped";
+    githubId = 47436522;
+    name = "deliciouslytyped";
+  };
```

Comment:
Also it would be nice if you would add another form of contact additionell to GitHub. Otherwise we might need to delete your entry if we can no longer reach you on that GitHub user and it wasn't updated.

Reply:
This was deliberate. If I'm no longer reachable, it's probably reasonable to delete.